node/actors/bft/src/testonly/run.rs

```diff
@@ -1,18 +1,41 @@
 use super::{Behavior, Node};
-use std::collections::HashMap;
+use anyhow::bail;
+use network::{io, Config};
+use rand::prelude::SliceRandom;
```

Comment:
nit `rand::seq::SliceRandom as _`. `prelude` is only for `use rand::prelude::*` and we don't use wildcard imports.